neuralop/models/fnogno.py

```diff
@@ -13,6 +13,98 @@
 
 
 class FNOGNO(nn.Module):
+    """FNOGNO: Fourier/Geometry Neural Operator 
+
+        Parameters
+        ----------
+        in_channels : _type_
+            number of input channels
+        out_channels : _type_
+            number of output channels
+        projection_channels : int, optional
+             number of hidden channels in embedding block of FNO. Defaults to 256.
+        gno_coord_dim : int, optional
+            dimension of GNO input data. Defaults to 3.
+        gno_coord_embed_dim : _type_, optional
+            dimension of embeddings of GNO coordinates. Defaults to None.
+        gno_radius : float, optional
+            radius parameter to construct graph. Defaults to 0.033.
+        gno_mlp_hidden_layers : list, optional
+            dimension of hidden MLP layers of GNO. Defaults to [512, 256].
+        gno_mlp_non_linearity : nn.Module, optional
+            nonlinear activation function between layers, by default F.gelu
+        gno_transform_type : str, optional
+            type of integral transform to apply in GNO. Defaults to 'linear'.
+            options: 'linear_kernelonly' 
+                        'linear' 
+                        'nonlinear_kernelonly' 
+                        'nonlinear'
+        gno_use_open3d : bool, optional
+            whether to use Open3D functionality. Defaults to False.
```

Comment:
Same, the doc should provide additional info, e.g. "If not, defaults to torch_scatter"